### PR TITLE
Gem

### PR DESCRIPTION
**Pull request by dannyvassallo**
@TSMMark this is working if you run `bin/console` from the shell in the project root. If you pass it the `VineDl.login_and_get_videos("email", "password")` it will download the videos in a download directory within the lib folder.

I want to make this work well for Vydia's stack particularly, so I'd need to figure out a way to set the default path to the root -- or let the user choose.

Any Idea on a good way to handle that?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,49 @@
...
-Create a `.env` modeled after `.env.example`.
+OR
 
-Add your Vine login credentials to the `.env` file.
+Install from github:
 
-```shell
-  ruby vine_dl.rb
+```ruby
+gem 'vine_dl', :git => 'git://github.com/dannyvassallo/vine_dl.git'
```
Comment (TSMMark) [inline README.md]: why not publish on rubygems?

Comment (TSMMark) [inline README.md]: nvm

**Comment by TSMMark** [comment]
The way you use this in an app like Vydia is, you don't use `VineDl.login_and_get_videos` at all. I only left that function in tact as to not break your original script. i.e., so you could continue to run it from the command line easily.

But that function is only a wrapper for the real in-app API, which is:

``` ruby
user = VineDl::User.new(username, password)
user.videos.each do |video|
  video.download("custom/path/to/wherever.mp4")
end
```

**Comment by TSMMark** [comment]
This approach will also support only downloading on the videos you want. The `VineDl::Video` wrapper class does not externalize any metadata yet, but it would be easy too.

For example, in the future you could do something like this:

``` ruby
user = VineDl::User.new(username, password)
recent_videos = user.videos.select { |video| video.created_at > 1.day.ago }
recent_videos.each do |video|
  video.download("custom/path/to/wherever.mp4")
end
```

**Comment by TSMMark** [comment]
PR looks fine tho. Ya don't have to wait for me to merge

**Comment by dannyvassallo** [comment]
@TSMMark haha i know i don't just like to get your input especially if you all are going to use it.

**Comment by dannyvassallo** [comment]
@TSMMark I'll update the readme to reflect your api usage and then merge.

**Pull request merged**